ui/helpers/constants/design-system.js

```diff
@@ -181,6 +181,7 @@ export const SIZES = {
   MD: 'md',
   LG: 'lg',
   XL: 'xl',
+  AUTO: 'auto',
```

Comment:
Worth adding a comment here? Something like
```suggestion
  AUTO: 'auto', // Used for Text and Icon components to inherit the parent elements font-size
```